js/main.js

```diff
@@ -282,56 +360,29 @@ function constructSampleHTML(samples) {
   const sampleCardHTML = samples
     .map((sample) => {
       return `
-            <a
-            class="group relative flex h-[280px] 2xl:h-[360px]  flex-col gap-6 rounded-xl overflow-hidden"
-            href="${sample.sampleUrl}"
-            ><div
-              class="absolute inset-0 h-[190px] 2xl:h-[240px]  rounded-xl overflow-hidden group-hover:h-full w-full duration-[250ms]"
-            >
+            <a class="group relative flex h-[280px] 2xl:h-[360px]  flex-col gap-6 rounded-xl overflow-hidden" href="${sample.sampleUrl}">
+            <div class="absolute inset-0 h-[190px] 2xl:h-[240px]  rounded-xl overflow-hidden group-hover:h-full w-full duration-250">
               <video
-              poster="${sample.poster}"
+              poster="${baseUrl}assets/${sample.id}.png"
               class="h-full w-full rounded-xl object-cover group-hover:scale-125 duration-1000"
-              src="${sample.video}"
+              src="${baseUrl}assets/${sample.id}.mp4"
               muted
               loop
               ></video>
               <div class="absolute top-[10px] right-[10px]">
               <div class="flex flex-col items-end flex-wrap gap-2">
-              ${
-                sample.model
-                  ? ` <div
-              class="flex rounded-md font-semibold bg-stone-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto"
-            >
-              ${sample.model}
-            </div>`
-                  : ``
-              }
-                  ${
-                    sample.webApi.length > 0
-                      ? ` <div
-                  class="flex rounded-md font-semibold bg-indigo-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto"
-                > ${sample.webApi.join("·")} · ${sample.framework}</div>`
-                      : ``
-                  }
-                  ${
-                    sample.supportHdw.length > 0
-                      ? ` <div
-                  class="flex rounded-md font-semibold bg-sky-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto"
-                >
-                  ${sample.supportHdw.join(" · ")}
-                </div>`
-                      : ``
-                  }
-
+              <div class="flex rounded-md font-semibold bg-sky-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto">
```

Comment:
Adjust the UI a bit, as info like model, webApi, framework and HW cover a lot of pictures and videos.